src/Ng/Jobs/CatalogUtility.cs

```diff
@@ -171,7 +180,17 @@ private static DateTime ForceUtc(DateTime date)
                     // This query string will ensure the package is not cached
                     // (e.g. on the CDN) and returns the "latest and greatest" package metadata.
                     var packageUri = Utilities.GetNugetCacheBustingUri(packageItem.ContentUri, entry.Key.ToString("O"));
-                    var response = await client.GetAsync(packageUri, cancellationToken);
+                    HttpResponseMessage response = null;
+                    try
+                    {
+                        response = await client.GetAsync(packageUri, cancellationToken);
```

Comment:
Have we considered increasing the HTTP timeout? Will that help? Is it a good idea? 

I don't understand what cases would lead to this sort of timeout.